packages/orchestrator-ui-components/src/components/WfoPageTemplate/WfoPageHeader/WfoPageHeader.tsx

```diff
@@ -18,22 +21,29 @@ import { WfoAppLogo } from '@/components/WfoPageTemplate/WfoPageHeader/WfoAppLog
 import { getWfoPageHeaderStyles } from '@/components/WfoPageTemplate/WfoPageHeader/styles';
 import { useOrchestratorTheme, useWithOrchestratorTheme } from '@/hooks';
 import { WfoLogoutIcon, WfoSideMenu } from '@/icons';
+import { ColorModes } from '@/types';
 
 export interface WfoPageHeaderProps {
     // todo: should be part of theme!
     navigationHeight: number;
     getAppLogo: (navigationHeight: number) => ReactElement;
     handleSideMenuClick: () => void;
     handleLogoutClick: () => void;
+    onThemeSwitch: (theme: EuiThemeColorMode) => void;
 }
 
+const ENABLE_THEME_SWITCH =
+    process.env.NEXT_PUBLIC_ENABLE_THEME_SWITCH || false;
```

Comment:
You could make this part of the orechestrator config to avoid using process.env in the client parts of the app